var/spack/repos/builtin/packages/py-scipy/package.py

```diff
@@ -200,14 +195,16 @@ def setup_build_environment(self, env):
     def install(self, spec, prefix):
         blas = spec["blas"].libs.names[0]
         lapack = spec["lapack"].libs.names[0]
-        # FIXME: MKL support doesn't work, why?
         if spec["blas"].name in ["intel-mkl", "intel-parallel-studio", "intel-oneapi-mkl"]:
             blas = "mkl-dynamic-lp64-seq"
         if spec["lapack"].name in ["intel-mkl", "intel-parallel-studio", "intel-oneapi-mkl"]:
             lapack = "mkl-dynamic-lp64-seq"
-
         if spec["blas"].name in ["blis", "amdblis"]:
             blas = "blis"
+        if blas == "armpl":
+            blas += "-dynamic-lp64-seq"
+        if lapack == "armpl":
+            lapack += "-dynamic-lp64-seq"
```

Comment:
The docs at https://developer.arm.com/downloads/-/arm-performance-libraries/get-started-with-armpl-free-version/single-page say that the correct name is `armpl_lp64`. And the link line in [this doc pagefor ILP64](https://developer.arm.com/documentation/101004/2210/General-information/Arm-Performance-Libraries-variant-using-64-bit-integer--INTEGER-8--arguments?lang=en) says `-larmpl_ilp64`. 

I can't find anything about pkg-config file names, but I wanted to check: are you sure this is right? If you could show a directory listing with all pkg-config files as shipped in ArmPL, that would be quite helpful. Both for review here, and for adding support in Meson for ArmPL, so we don't need this patch anymore in the future.

Reply:
I should have been more clear about the last sentence: "Depends on https://github.com/spack/spack/pull/34978". Please take a look for reference.

ArmPL ships pkg-config files under `armpl_22.1_gcc-11.2/pkgconfig` and they don't have the `.pc` suffix. I am in contact with ARM to change that in the future.

Reply:
Yes, I saw that and am following along with that PR. But it doesn't tell me what the file names are. Can you show the output of `ls armpl_22.1_gcc-11.2/pkgconfig`?

Reply:
Directory content (after applying https://github.com/spack/spack/pull/34978):
```
[ec2-user@ip-10-7-99-193 ~]$ ls $(find $(spack location -i armpl-gcc) -name pkgconfig)
armpl-dynamic-ilp64-omp             armpl-Fortran-static-ilp64-omp
armpl-dynamic-ilp64-omp.pc          armpl-Fortran-static-ilp64-omp.pc
armpl-dynamic-ilp64-seq             armpl-Fortran-static-ilp64-seq
armpl-dynamic-ilp64-seq.pc          armpl-Fortran-static-ilp64-seq.pc
armpl-dynamic-lp64-omp              armpl-Fortran-static-lp64-omp
armpl-dynamic-lp64-omp.pc           armpl-Fortran-static-lp64-omp.pc
armpl-dynamic-lp64-seq              armpl-Fortran-static-lp64-seq
armpl-dynamic-lp64-seq.pc           armpl-Fortran-static-lp64-seq.pc
armpl-Fortran-dynamic-ilp64-omp     armpl-static-ilp64-omp
armpl-Fortran-dynamic-ilp64-omp.pc  armpl-static-ilp64-omp.pc
armpl-Fortran-dynamic-ilp64-seq     armpl-static-ilp64-seq
armpl-Fortran-dynamic-ilp64-seq.pc  armpl-static-ilp64-seq.pc
armpl-Fortran-dynamic-lp64-omp      armpl-static-lp64-omp
armpl-Fortran-dynamic-lp64-omp.pc   armpl-static-lp64-omp.pc
armpl-Fortran-dynamic-lp64-seq      armpl-static-lp64-seq
armpl-Fortran-dynamic-lp64-seq.pc   armpl-static-lp64-seq.pc
```

Reply:
Great, thank you! I'll plan to add support for those names in Meson (xref https://github.com/mesonbuild/meson/pull/10921).